<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>DirectoryServicesWidgetBase</class>
 <widget class="QWidget" name="DirectoryServicesWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Directory Services Configuration</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="x500LA" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>X.&amp;500 directory services:</string>
     </property>
     <property name="buddy" >
      <cstring>x500LV</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="Q3ListView" name="x500LV" >
       <property name="toolTip" >
        <string>Select Directory Services to Use Here</string>
       </property>
       <property name="whatsThis" >
        <string>&lt;qt>
&lt;h1>X.500 Directory Services&lt;/h1>
You can use X.500 directory services to retrieve certificates and certificate revocation lists that are not saved locally. Ask your local administrator if you want to make use of this feature and are unsure which directory service you can use.
&lt;p>
If you do not use a directory service, you can still use local certificates.
&lt;/qt></string>
       </property>
       <property name="allColumnsShowFocus" >
        <bool>true</bool>
       </property>
       <column>
        <property name="text" >
         <string>Server Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Port</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Base DN</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>User Name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Password</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QToolButton" name="upButton" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="downButton" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>51</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addServicePB" >
       <property name="toolTip" >
        <string>Click to add a service</string>
       </property>
       <property name="whatsThis" >
        <string>&lt;qt>
&lt;h1>Add a Directory Service&lt;/h1>
By clicking this button, you can select a new directory service to be used for retrieving certificates and CRLs. You will be asked for the server name and an optional description.
&lt;/qt></string>
       </property>
       <property name="text" >
        <string>&amp;Add Service...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeServicePB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="toolTip" >
        <string>Click to remove the currently selected service</string>
       </property>
       <property name="whatsThis" >
        <string>&lt;qt>
&lt;h1>Remove Directory Service&lt;/h1>
By clicking this button, you can remove the currently selected directory service in the list above. You will have a chance to rethink your decision before the entry is deleted from the list.
&lt;/qt></string>
       </property>
       <property name="text" >
        <string>&amp;Remove Service</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>47</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends></extends>
   <header>q3listview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
